spec/acceptance/suites/default/00_simp_server_install_spec.rb

```diff
@@ -53,8 +56,16 @@ def create_r10k_puppetfile(simp_core_puppetfile)
       on(master, 'puppet resource service puppetserver ensure=running')
     end
 
-    it 'should enable trusted_server_facts' do
-      on(master, 'puppet config --section master set trusted_server_facts true')
+    it 'should do some misc configuration' do
```

Comment:
Can we split this into two `it` statements with appropriate titles so that it's easier to tell from the screen output if something failed?